adoptopenjdk-api-v3-frontend/src/test/kotlin/net/adoptopenjdk/api/AssetsResourceFeatureReleasePathTest.kt

```diff
@@ -63,43 +56,40 @@ class AssetsResourceFeatureReleasePathTest : AssetsPathTest() {
 
     @Test
     fun sortOrderASCIsHonoured() {
-
-        val times = getTimestamps(SortOrder.ASC)
-
-        times
-                .fold(LocalDateTime.MIN, { previous, updatedAt ->
-                    assertTrue("${previous} is after ${updatedAt}", updatedAt.isAfter(previous) || updatedAt.equals(previous))
-                    updatedAt
+        getReleases(SortOrder.ASC)
+                .fold(null, { previous: Release?, next: Release ->
+                    if (previous != null) {
+                        if (previous.version_data.compareTo(next.version_data) > 0) {
```

Comment:
What happens with =?

Reply:
if we ever release 2 releases with the same version, and include it in the test set, then this test fails